client/my-sites/stats/stats-purchase/stats-commercial-upgrade-slider/index.tsx

```diff
@@ -36,12 +40,14 @@ function getStepsForTiers( tiers: StatsPlanTierUI[] ) {
 		if ( typeof tier.price === 'string' ) {
 			price = tier.price;
 		}
-		// Views can be a number or a string so address that.
+		// View should be a number but the current mock data
+		// includes a string for the final tier.
+		// Special case that scenario for now.
 		let views = '';
-		if ( typeof tier.views === 'string' ) {
-			views = tier.views;
-		} else if ( typeof tier.views === 'number' ) {
+		if ( typeof tier.views === 'number' ) {
 			views = formatNumber( tier.views );
+		} else if ( typeof tier.views === 'string' ) {
```

Comment:
The `tier.views` seems always to be a number or null rather than a string.

Reply:
Thanks Dognose. Looks good!